scripts/setup-pay-per-crawl.sh

```
@@ -0,0 +1,79 @@
+echo "1. Cloudflare account (free tier is fine)"
+echo "2. Your domain added to Cloudflare"
+echo "3. API token from Cloudflare dashboard"
+echo ""
+
+echo -e "${YELLOW}🚀 Step 1: Sign up for Pay-Per-Crawl${NC}"
+echo "Visit: https://www.cloudflare.com/paypercrawl-signup/"
+echo "Press Enter when you've signed up..."
+read
+
+echo -e "${YELLOW}🔧 Step 2: Install dependencies${NC}"
+cd landing-pages/sudoku-for-seniors
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)